WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/widget/ViewsWidgetListViewModel.kt

```diff
@@ -0,0 +1,106 @@
+@file:JvmName("StatsViewsWidgetConfigureViewModelKt")
+
+package org.wordpress.android.ui.stats.refresh.lists.widget
+
+import androidx.annotation.LayoutRes
+import kotlinx.coroutines.runBlocking
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.stats.LimitMode.Top
+import org.wordpress.android.fluxc.model.stats.time.VisitsAndViewsModel.PeriodData
+import org.wordpress.android.fluxc.network.utils.StatsGranularity.DAYS
+import org.wordpress.android.fluxc.store.SiteStore
+import org.wordpress.android.fluxc.store.stats.time.VisitsAndViewsStore
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ValueItem.State.NEGATIVE
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ValueItem.State.NEUTRAL
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ValueItem.State.POSITIVE
+import org.wordpress.android.ui.stats.refresh.lists.sections.granular.usecases.OverviewMapper
+import org.wordpress.android.ui.stats.refresh.lists.widget.StatsViewsWidgetConfigureViewModel.Color
+import org.wordpress.android.ui.stats.refresh.utils.StatsDateFormatter
+import org.wordpress.android.viewmodel.ResourceProvider
+import java.util.Date
+import javax.inject.Inject
+
+const val LIST_ITEM_COUNT = 7
+
+class ViewsListViewModel
```

Comment:
To stay consistent with the rest of the filenames, maybe we should call this class something like `ViewsWidgetListViewModel`. WDYT?

Reply:
Sounds good 👍